packages/openapi3/test/primitive-types.test.ts

```diff
@@ -115,6 +115,30 @@ describe("openapi3: primitives", () => {
       });
     });
 
+    it("scalar used as a default value", async () => {
```

Comment:
can you try more with one of the scalar that we have as well as making sure that `@encode` works